firmware/tunerstudio/rusefi.input

```diff
@@ -69,10 +69,6 @@ enable2ndByteCanID = false
 ; CONFIG_DEFINITION_END
 	
 
-[SettingContextHelp]
-
-	fuelAlgorithm = "See also http://rusefi.com/fuel"
```

Comment:
(I've modified the other PR but same question) why are we removing SettingContextHelp?

Reply:
Redundancy:
![image](https://user-images.githubusercontent.com/15104768/37579972-eb8ea1da-2b49-11e8-987b-e805800d18d8.png)

The new version is more explanatory:
![image](https://user-images.githubusercontent.com/15104768/37580013-2d0309bc-2b4a-11e8-8cba-f984ec4c9a53.png)

Besides, we've mastered an integrated help system in TunerStudio, and we can add any info or web-links there also:
![image](https://user-images.githubusercontent.com/15104768/37580127-bcdf9d7a-2b4a-11e8-892b-c59b308ba38c.png)